common/fw-update-helper.cpp

```diff
@@ -236,7 +236,9 @@ namespace rs2
                                 }
                             }
                         }
-                        catch (...) {}
+                        catch (std::exception &e) {
+                            LOG_ERROR("Exception caught in process-flow: " << e.what() << "!!!!");
```

Comment:
1. Can the section throw non-`std::exception` class ? If yes - then an extra `catch(...)` clause will be needed
2. For clarity - ".. in **FW Update** process flow". No need for exclamation.